app/src/main/java/org/systers/mentorship/view/fragments/MembersFragment.kt

```diff
@@ -117,13 +117,15 @@ class MembersFragment : BaseFragment() {
                         if (!isRecyclerView) {
                             rvMembers.apply {
                                 layoutManager = LinearLayoutManager(context)
+
```

Comment:
don't need this change.

Reply:
I actually didn't get it, Are you asking to keep the code writing format/syntax to be as earlier (new line, extra spaces) or something else?
In all three comments?

Reply:
yes

Reply:
got it, On it :)